charts/poverty_num_min_china/poverty_num_min_china.css

```diff
@@ -0,0 +1,25 @@
+/* Please use always the chart id as primary selector to not interfer with other parts */
+
+/* Adjusting chart tools */
+
+#chart-demo_chart h1{
```

Comment:
This is still the wrong identifier for this chart. Either change it or delete it :)

Reply:
Thanks for the review :)
I changed both points of yours. On the identifier I chose to use the same css as inequality_gdp_nrt_sth.css does since this has been my inspiring chart